static/app/views/alerts/wizard/index.tsx

```diff
@@ -180,19 +180,21 @@ class AlertWizard extends Component<Props, State> {
             <WizardBody>
               <WizardOptions>
                 <Styledh2>{t('Errors')}</Styledh2>
-                {AlertWizardOptions.map(({categoryHeading, options}, i) => (
-                  <OptionsWrapper key={categoryHeading}>
-                    {i > 0 && <Styledh2>{categoryHeading}</Styledh2>}
-                    <RadioPanelGroup
-                      choices={options.map(alertType => {
-                        return [alertType, AlertWizardAlertNames[alertType]];
-                      })}
-                      onChange={this.handleChangeAlertOption}
-                      value={alertOption}
-                      label="alert-option"
-                    />
-                  </OptionsWrapper>
-                ))}
+                {getAlertWizardCategories(organization).map(
+                  ({categoryHeading, options}, i) => (
+                    <OptionsWrapper key={categoryHeading}>
+                      {i > 0 && <Styledh2>{categoryHeading}</Styledh2>}
+                      <RadioPanelGroup
+                        choices={options.map(alertType => {
+                          return [alertType, AlertWizardAlertNames[alertType]];
```

Comment:
nit: should this const have the first letter lower case? 
```suggestion
                          return [alertType, alertWizardAlertNames[alertType]];
```

Reply:
there are lots of constants like this in that file, I didn't want to go on a refactoring spree because this PR would get messy